src/olympia/scanners/tests/test_admin.py

```diff
@@ -263,7 +263,7 @@ def test_list_queries(self):
         deleted_addon.delete()
 
         with self.assertNumQueries(13):
-            # 14 queries:
+            # 13 queries:
```

Comment:
The breakdown below needs updating (?)

Reply:
Nah, I checked it, it was correct, this comment was just out of date.